### PR TITLE
Er collab

### PR DESCRIPTION
**Pull request by nkrabben**

### CODE REVIEW

**Comment by arhonemus** [comment]
[Review PR #26](https://trello.com/c/oXLn8aeV/45-review-pr-26)

File: project-management/electronic-records-steps-collaboration.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+# Accessioning  
+* Contact Digital Archives when digital media are in an accessioned collection. 
+
+# Collection Assignment
+* Contact Digital Archives when you are assigned a collection with digital media.  
+* Discuss your time frame for processing the collection.  
+* Find out whether media has been transferred and how that will affect your processing schedule.  
+* Discuss whether you would like to appraise/survey hard drive directories before transfer.  
+
+### Survey Directories (DRAFT)
```
Comment (dreyjo) [inline project-management/electronic-records-steps-collaboration.md]: How do we want to word drafts/processes that may change? Potentially a label?

File: accessioning/inventorying-digital-media-accession.md [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+    * Enter the serial number if the media object is a hard disk drive. Leave blank for other media types.  
+
+| Label transcript is an optional field. If you want to transcript a label you can enter text as it appears on the label. You can enter ```[illegible]``` for words you can't decipher.|  
+
+An example of a completed record for separation:
+
+![](/digitalmediaseparation/media/image1.png)
+
+Deprecated
+{: .label .label-red }
```
Comment (dreyjo) [inline accessioning/inventorying-digital-media-accession.md:120]: Is it possible for this label to be in line with the section 5 header or below it? Just so it's closer to what is being deprecated.

File: accessioning/digitalmediaseparation.md [hunk 3]
```diff
@@ -105,6 +105,8 @@ An example of a completed record for separation:
 
 ![](/digitalmediaseparation/media/image1.png)
 
+Deprecated
+{: .label .label-red }
```
Comment (dreyjo) [inline accessioning/digitalmediaseparation.md:117]: Is it possible for this label to be in line with the section 5 header or below it? Just so it's closer to what is being deprecated.

**Review by arhonemus — APPROVED** [review]

**Review by arhonemus — CHANGES_REQUESTED** [review]
Label changes

**Review by arhonemus — CHANGES_REQUESTED** [review]
label changes

**Review by arhonemus — CHANGES_REQUESTED** [review]
labels

**Review by arhonemus — CHANGES_REQUESTED** [review]
label

**Review by arhonemus — APPROVED** [review]
@dreyjo The label changes now appear in the pull request. (The previous comments I made failed to sync the changes.)

**Comment by nkrabben** [comment]
Label solution looks good. I'm merging this.

**Pull request merged**